units:
  boots:
  - name: S2iThothUbi8Py39v0300devVuln0QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:2717' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:2717
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:2717 - see https://access.redhat.com/errata/RHSA-2021:2717
  - name: S2iThothUbi8Py39v0300devVuln1QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3582 - see https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0300devVuln2QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3816 - see https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0300devVuln3QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3058 - see https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0300devVuln4QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3075 - see https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0300devVuln5QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3576 - see https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0300devVuln6QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3074 - see https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0300devVuln7QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3666 - see https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0300devVuln16QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3057 - see https://access.redhat.com/errata/RHSA-2021:3057
  - name: S2iThothUbi8Py39v0300devVuln20QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3073 - see https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0300devVuln21QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3623 - see https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0310Vuln0QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3058 - see https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0310Vuln1QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3816 - see https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0310Vuln2QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3576 - see https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0310Vuln4QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3074 - see https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0310Vuln5QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3666 - see https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0310Vuln8QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3075 - see https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0310Vuln11QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3057 - see https://access.redhat.com/errata/RHSA-2021:3057
  - name: S2iThothUbi8Py39v0310Vuln13QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3582 - see https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0310Vuln17QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3073 - see https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0310Vuln18QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3623 - see https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0320Vuln0QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.32.0
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3816 - see https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0311Vuln0QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3074 - see https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0311Vuln1QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3073 - see https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0311Vuln2QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3623 - see https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0311Vuln3QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3666 - see https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0311Vuln4QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3058 - see https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0311Vuln7QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3582 - see https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0311Vuln8QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3075 - see https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0311Vuln11QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3576 - see https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0311Vuln12QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3816 - see https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0311Vuln24QuaySecurityErrorBoot
    type: boot
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - security
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: ERROR
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
      not_acceptable: >-
        The base image used has a CVE RHSA-2021:3057 - see https://access.redhat.com/errata/RHSA-2021:3057
  wraps:
  - name: S2iThothUbi8Py39v0300devVuln0QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:2717' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:2717
  - name: S2iThothUbi8Py39v0300devVuln1QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0300devVuln2QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0300devVuln3QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0300devVuln4QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0300devVuln5QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0300devVuln6QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0300devVuln7QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0300devVuln16QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
  - name: S2iThothUbi8Py39v0300devVuln20QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0300devVuln21QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.30.0-dev
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0310Vuln0QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0310Vuln1QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0310Vuln2QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0310Vuln4QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0310Vuln5QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0310Vuln8QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0310Vuln11QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
  - name: S2iThothUbi8Py39v0310Vuln13QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0310Vuln17QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0310Vuln18QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0320Vuln0QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.32.0
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0311Vuln0QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3074' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3074
  - name: S2iThothUbi8Py39v0311Vuln1QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3073' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3073
  - name: S2iThothUbi8Py39v0311Vuln2QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3623' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3623
  - name: S2iThothUbi8Py39v0311Vuln3QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3666' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3666
  - name: S2iThothUbi8Py39v0311Vuln4QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3058' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3058
  - name: S2iThothUbi8Py39v0311Vuln7QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3582' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3582
  - name: S2iThothUbi8Py39v0311Vuln8QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3075' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3075
  - name: S2iThothUbi8Py39v0311Vuln11QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3576' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3576
  - name: S2iThothUbi8Py39v0311Vuln12QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3816' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3816
  - name: S2iThothUbi8Py39v0311Vuln24QuaySecurityWarningWrap
    type: wrap
    should_include:
      adviser_pipeline: true
      recommendation_types:
      - latest
      - performance
      - stable
      - testing
      runtime_environments:
        base_images:
        - quay.io/thoth-station/s2i-thoth-ubi8-py39:v0.31.1
    run:
      stack_info:
      - type: WARNING
        message: >-
          Found vulnerability 'RHSA-2021:3057' in the base image used
        link: https://access.redhat.com/errata/RHSA-2021:3057
